hq_superset/tests/test_utils.py

```diff
@@ -122,7 +122,4 @@ def _ensure_platform_roles_exist(self, sm):
 
     @staticmethod
     def _to_permissions_response(can_write, can_read, roles):
-        return {
-            "can_write": can_write,
-            "can_read": can_read,
-        }, roles
+        return can_read, can_write, roles
```

Comment:
This method seems redundant, but it's providing a more clear structure to the expected results in the test. 

For instance, instead of doing this in the test
```
  get_domain_access_mock.return_value = (False, False ,["sql_lab", "dataset_editor"])
```
I can do this
```
get_domain_access_mock.return_value = self._to_permissions_response(
    can_write=False,
    can_read=False,
    roles=["sql_lab", "dataset_editor"],
)
```
Using a method like this makes it more clear what each value means.